sims/verilator/Makefile

```diff
@@ -47,67 +47,133 @@ debug: $(sim_debug)
 include $(base_dir)/common.mk
 
 #########################################################################################
-# verilator binary and flags
+# verilator-specific user-interface variables and commands
```

Comment:
I wonder if this should be pulled into a `verilator.mk` a la `vcs.mk`. I don't think we currently use it elsewhere but perhaps we will at some point, and in the meantime it adds some nice parallelism to the two currently supported simulators.

Reply:
I agree the parallelism would be nice. If we do that, I would like to see verilator.mk and vcs.mk moved into sims/verilator and sims/vcs

Reply:
The reason `vcs.mk` is in the top-level is because its used by `vlsi`

Reply:
Yes, but `vlsi` can just look in `../sims/vcs` instead?

Reply:
Sure I guess that works as well. I don't have a strong preference. But I do feel like if we end up with a `sim.mk` `vcs.mk` and `verilator.mk` it would be slightly annoying to have them all in different directories.

Reply:
Will save this discussion for the PR that solves #653